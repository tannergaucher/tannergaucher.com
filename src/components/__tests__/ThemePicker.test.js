import React from 'react'
import { render } from 'react-testing-library'
import renderer from 'react-test-renderer'
import ThemePicker from '../ThemePicker'

describe('<ThemePicker/>', () => {
  it('renders correctly and matches the snapshot', () => {
    // props here
    const tree = renderer.create(<ThemePicker />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders an svg image', () => {})
})
