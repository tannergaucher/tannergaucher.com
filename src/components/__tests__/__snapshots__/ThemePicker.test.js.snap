// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemePicker/> renders correctly and matches the snapshot 1`] = `
<div
  className="sc-bdVaJa DlJkq"
>
  <img
    alt="moon"
    style={
      Object {
        "height": "20px",
        "width": "20px",
      }
    }
  />
</div>
`;
